diff --git a/fast_planner/plan_manage/src/kino_replan_fsm.cpp b/fast_planner/plan_manage/src/kino_replan_fsm.cpp
--- a/fast_planner/plan_manage/src/kino_replan_fsm.cpp
+++ b/fast_planner/plan_manage/src/kino_replan_fsm.cpp
@@ -74,1 +74,1 @@
-    end_pt_ << msg->poses[0].pose.position.x, msg->poses[0].pose.position.y, 1.0;
+    end_pt_ << msg->poses[0].pose.position.x, msg->poses[0].pose.position.y, msg->poses[0].pose.position.z;
